index.html

```diff
@@ -136,9 +169,9 @@ <h3 class="subtitle">Frameworks</h3>
                 <li class="aboutMeListItem">Capybara</li>
                 <li class="aboutMeListItem">Selenium</li>
             </ul>
-        </section>
-        <section class="aboutMeCard flex">
-            <img src="images/IconSkills.svg" alt="Languages">
+        </div>
+        <div class="aboutMeCard flex">
+            <img src="./images/iconskills.svg" alt="Tool skills" />
             <h3 class="subtitle">Skills</h3>
             <ul>
                 <li class="aboutMeListItem">Codekit</li>
```

Comment:
- [ ] Your screen is almost responsive to all screen sizes except between `769px` to `1030`, Here I can see some contents are overflowing and I am required to use `Horizontal scrolling` this is not a good practice according to web accessibility. Also, please check for screen size between `375px` and  `400px` here also some contents are slightly overflowing.
![Capture-2](https://user-images.githubusercontent.com/69077061/200628026-6e61bf8b-c553-4424-9d0a-1374c44ae8fb.PNG)